functions/Get-DbaServerInstallDate.ps1

```diff
@@ -0,0 +1,181 @@
+function Get-DbaServerInstallDate
+{
+<#
+.SYNOPSIS
+Returns the install date of SqlServer and Windows Server, depending on what is passed. 
+	
+.DESCRIPTION
+By default, this command returns for each SQL Server instance passed in:
+SQL Instance install date, formatted as a string
+Hosting Windows server install date, formatted as a string
+	
+.PARAMETER SqlServer
+The SQL Server that you're connecting to.
+
+.PARAMETER SqlCredential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER WindowsCredential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER SqlOnly
```

Comment:
I don't think an include / exclude param fits this since there are only two instances. Also, I think that if the end user is using 2 -*only switches it should be caught earlier in the flow and then the function is stopped to for the end user to correct. 

@potatoqualitee?